tests.py

```diff
@@ -1,4 +1,5 @@
 # Pytest doc: https://docs.pytest.org/en/latest/getting-started.html#create-your-first-test
+
```

Comment:
No test? :/

Reply:
I won't write tests for `shorten_dataset_proportion`, the fixtures I wrote are in the other branch #190